### DISCUSSION TITLE
Contribution Docs - Getting stuck on `npm install`. @alpinejs/history not found.

### DISCUSSION THREAD

**Discussion by citricguy** [livewire/livewire#6228] [1. Help] [answered]
I'm running through the instructions at [https://laravel-livewire.com/docs/2.x/contribution-guide](https://laravel-livewire.com/docs/2.x/contribution-guide) but keep getting hung up on the npm install.

Here is the error I'm getting:
```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@alpinejs%2fhistory - Not found
npm ERR! 404
npm ERR! 404  '@alpinejs/history@^3.10.5' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

Where can I find `'@alpinejs/history@^3.10.5'` or is there something obvious I'm missing?

**Comment by PhiloNL** [comment, marked as answer]
Do you want to make adjustments to the Livewire JS library?
If not, you can skip the `npm install` step.

Otherwise, you will have to clone the Alpine repository and use `npm link` to get the `history` package.

**Reply by joshhanley** [reply]
@citricguy yeah I agree with @PhiloNL, for the moment, don't install NPM assets as you have to jump through some hoops to get it to work at the moment.

**Reply by citricguy** [reply]
Thank you guys for the feedback, everything I was experimenting with doesn't involve JS. :)
